### PR TITLE
Fix past tense commit messages by present imperative ones

### PR DESCRIPTION
**Pull request by matiasbeckerle**
Hey folks, could you review this one? Thanks in advance.

## Background

While @lucianaDilizio was reading this documentation found some inconsistencies between rule 1.1 (using present imperative for commit messages) and other rules/examples.

This PR aims to fix those inconsistencies in order to avoid confusion and be precise about what is expected.

## Changes

- Fixed past tense commit messages by present imperative ones.

### CODE REVIEW

**Comment by matiasbeckerle** [comment]
Suggested reviewers: @AlphaGit, @andresmoschini, @lucianaDilizio. I can't add reviewers to the PR so I'm posting this comment.

File: git-workflow/conventions.md [hunk 1]
```diff
@@ -168,12 +168,12 @@ Make sure the message of the last commit has the ticket number of the tracking c
...
 Good examples:
 
-- `[CFP-114] Removed unused event handlers`
-- `[CFP-110] Updated main page styles`
+- `[CFP-114] Remove unused event handlers`
+- `[CFP-110] Update main page styles`
 
-**5.2.1.** It is ok to have multiple commits for the same ticket. It may have taken several tries to achieve it, or it may just be very complicated for a single commit. Prioritize readability and traceability over having less commits.
\ No newline at end of file
+**5.2.1.** It is ok to have multiple commits for the same ticket. It may have taken several tries to achieve it, or it may just be very complicated for a single commit. Prioritize readability and traceability over having less commits.
```
Comment (andresmoschini) [inline git-workflow/conventions.md]: It seems that some invisible character have been modified here, what do you thing about revert this change?

Comment (matiasbeckerle) [inline git-workflow/conventions.md]: Hahaha you're a fanatic. Don't worry. In this case was an intended extra new line at the end of file. File had 179 lines and I added an extra 180 one. I can revert it anyways if you don't want to have those. I also can add an .editorconfig file.

Comment (andresmoschini) [inline git-workflow/conventions.md]: No, no, it was only a suggestion, I were only trying to avoid unnecessary changes that could obfuscate repository history.

If it is a desired change, I am fine.

**Comment by matiasbeckerle** [comment]
Actually... I first though the same @lucianaDilizio but then I realized that the "bad example" is something that only applies for 1.1 because rules after 1.1 already assumes that present imperative is the rule. So I believe it makes more sense to have it in this way. The remaining rules are for other topics so we can base each example using only the corrupted rule but apply the others in order to have the reader focused only in the corrupted rule. Otherwise reader will have all the wrong rules in the bad example and will not be able to figure it out which one is the wrong one.

What do you think?

**Comment by lucianaDilizio** [comment]
> Actually... I first though the same @lucianaDilizio but then I realized that the "bad example" is something that only applies for 1.1 because rules after 1.1 already assumes that present imperative is the rule. So I believe it makes more sense to have it in this way. The remaining rules are for other topics so we can base each example using only the corrupted rule but apply the others in order to have the reader focused only in the corrupted rule. Otherwise reader will have all the wrong rules in the bad example and will not be able to figure it out which one is the wrong one.
>
>What do you think?

I think that it's ok. Like do you say, the present imperative is assumed in the other rules and if you left it bad in that cases, it could generate confusion.

File: git-workflow/conventions.md [hunk 2]
```diff
@@ -34,46 +34,46 @@ Some tools, including console and GitHub are optimized for message sizes of abou
...
 
 Good examples:
 
 - Example 1:
 
     ```
-    Refactored mainComponent.js
+    Refactor mainComponent.js
 
     Removed manual calls to $scope.$digest() to avoid events being invoked even when they already where unhooked.
```
Comment (lucianaDilizio) [inline git-workflow/conventions.md]: I think that this example needs to be fixed too.

Comment (matiasbeckerle) [inline git-workflow/conventions.md]: Good catch. I believe that case is a neutral zone because is not specified. My taste is to leave the decision to the developer: chose whatever feels better in the scenario. When I describe a particular commit I feel that past tense is valid but... is my opinion 😄.

Thoughts?

**Comment by matiasbeckerle** [comment]
Bump. Maybe @andresmoschini you find a minute to provide your blessing on this one? 😄

**Comment by andresmoschini** [comment]
I think that it is a good change, you are doing a nice and lovely work finding details and fixing them, but I have not authority to _approve_ pull requests in this repository (and maybe in any one). 

I feel that _approving_ a pull request is a little pretentious thing, so, please allow me to avoid that. 

You know that you always have my blessing, so, I have already read the changes and I do not have anything else to said.

**Comment by matiasbeckerle** [comment]
Looks mergeable. What do you think @AlphaGit?

**Comment by AlphaGit** [comment]
@matiasbeckerle Absolutely! Feel free to go ahead.

**Comment by matiasbeckerle** [comment]
I don't have permissions for doing that @AlphaGit :P

**Comment by AlphaGit** [comment]
@matiasbeckerle My bad, I thought you did.

**Comment by matiasbeckerle** [comment]
np dude! @AlphaGit thanks!
